sdk/core/Azure.Core/src/ClientOptions.cs

```diff
@@ -99,6 +99,11 @@ public HttpPipelineTransport Transport
         /// </summary>
         public HttpPipelinePolicy? RetryPolicy { get; set; }
 
+        /// <summary>
+        /// The strategy to use for all delays.
+        /// </summary>
+        public DelayStrategy? Delay { get; set; }
```

Comment:
Is it only delays in the retry policy?  Would it make sense to put it on RetryOptions instead of at the top level?  Would it make sense to call it RetryDelayStrategy since Delay is [pretty heavily used](https://learn.microsoft.com/en-us/dotnet/api/?term=delay) in .NET APIs?

Reply:
No this could be used for more than just retries, e.g. LROs.

Reply:
I think we also want to avoid putting it on RetryOptions because that is meant to be protocol-agnostic. We could probably have the dependency go the other way - you can pass in a RetryOptions when constructing the Delay.